src/components/user/tour/create-tour/create-tour.css

```diff
@@ -0,0 +1,91 @@
+.create-tour-modal .create-tour-container {
```

Comment:
No need to create separate CSS files.
Instead of writing custom CSS, you could use `Dialog` component from Material UI: https://material-ui.com/components/dialogs/ 

CSS files from this PR should be removed